praw/models/reddit/subreddit.py

```diff
@@ -2204,8 +2205,9 @@ def __iter__(self):
                print(emoji)
 
         """
-        response = self.subreddit._reddit.get(
-            API_PATH['emoji_list'].format(subreddit=self.subreddit),
-            params={'unique': self.subreddit._reddit._next_unique})
-        for emoji_name in response[self.subreddit]:
-            yield Emoji(self.subreddit._reddit, self.subreddit, emoji_name)
+        response = self.subreddit.fullname._reddit.get(
```

Comment:
`response = self.subreddit.fullname._reddit.get(` should still be `response = self.subreddit._reddit.get(`.